### PR TITLE
plugins v0.7.2

### PR DESCRIPTION
**Pull request by migueltol22**
# TL;DR
Bump Plugins that includes support for batchScheduler support in spark tasks

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [x] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -240,6 +240,8 @@ github.com/flyteorg/flyteidl v0.21.4 h1:gtJK5rX2ydLAo2xLRHHznOSLuLHrRRdXDbpEAlxl
 github.com/flyteorg/flyteidl v0.21.4/go.mod h1:576W2ViEyjTpT+kEVHAGbrTP3HARNUZ/eCwrNPmdx9U=
 github.com/flyteorg/flyteplugins v0.7.1 h1:YdCEQtdPeol7u6LkopGTIfPLAhy3KcclQa+DZFauK8w=
 github.com/flyteorg/flyteplugins v0.7.1/go.mod h1:kOiuXk1ddIEVSPoHcc4kBfVQcLuyf8jw3vWJT2Was90=
+github.com/flyteorg/flyteplugins v0.7.2 h1:WLftq5+/RYt3HLc/mjRprqexot3brlMJHMH1yS7jee4=
```
Comment (EngHabu) [inline go.sum:243]: Can you run `go mod tidy` to clean up the extra imports here...

Comment (migueltol22) [reply go.sum:241]: done!

**Review by EngHabu — APPROVED** [review]
